components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -277,10 +279,9 @@ public SessionContextDO getSessionContextData(String key, String type) {
             resultSet = preparedStatement.executeQuery();
             if(resultSet.next()) {
                 String operation = resultSet.getString(1);
-                long timestamp = resultSet.getLong(3)/1000000;
+                long nanoTime = resultSet.getLong(3);
                 if ((OPERATION_STORE.equals(operation))) {
-                    return new SessionContextDO(key, type, getBlobObject(resultSet.getBinaryStream(2)),
-                            new Timestamp(timestamp));
+                    return new SessionContextDO(key, type, getBlobObject(resultSet.getBinaryStream(2)), nanoTime);
```

Comment:
Would this cause serialisation error with the old data?

Since earlier SessionContextDO had timestamp attribute and now having a long attribute..